### ISSUE TITLE
No sound on alarm when phone is muted

### ISSUE THREAD

**Issue by fluks** [SimpleMobileTools/Simple-Clock#100]
* Samsung Galaxy A3 (2017)
* Android 8.0.0
* Simple Clock 4.3.0

Ok, the subject is contradictory, I know. On my old phone with its stock clock app, when I set an alarm and I muted the phone, it would still play audio alarm. The same happens with my new phone's stock clock app. I mute my phone for the night, because I don't want to wake up to a phone call, but I want to wake up to an alarm. With Simple Clock the audio is not played in this case. I can't seem to mute only the ringtone volume and leave notifications volume as is in this phone.

Is it possible to override the system's volume settings in the app somehow? It must be.

**Comment by tibbi** [comment]
Try setting a different audio stream for reminders in the app settings, maybe with that you can figure it out somehow. Many things related to reminders depend on system settings, it just isnt something that I can make always work properly.

**Comment by fluks** [comment]
I installed Clock+ from F-Droid and it has a setting for an alarm sound and it is able to play an alarm when the phone is otherwise muted. I couldn't find that setting anywhere from my phone's system settings, weird. And if I muted the phone, then set the alarm volume in Clock+ to something audible and set an alarm in Simple Clock, the alarm didn't play a sound.

I think I'm going to look how Clock+ does that.
